main.go

```diff
@@ -259,6 +297,19 @@ func createCluster(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func getScaleNodeRole(role string) (bool, bool, error) {
```

Comment:
I'd prefer moving this logic inside the RunE func. It does little good to wrap it with a func and is hard for the caller to know what the two returned boolean vars are